wallets/rn_cli_wallet_068_5/src/components/W3WText.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {StyleSheet, Text} from 'react-native';
+import {FONT_COLORS} from '../utils/Theming';
+
+interface IW3WTextProps {
+  value: string;
+  type?: string;
+  color?: string;
+}
+
+/*
+  @notice W3WText Component (Unfinished / not swapped out for all TEXT components)
+  @params value, color
```

Comment:
Unfinished comment (`Color:`), `@params` is misaligned with the current props. 

Adding contextual comments (i.e. answering "why?" questions) is great but I don't think there's a need for full Solidity-style natspec, especially with WIP like this since it will become misaligned/misleading if not kept in shape.